models/src/main/scala/coop/rchain/models/rholang/sorter/GroundSortMatcher.scala

```diff
@@ -17,59 +17,6 @@ private[sorter] object GroundSortMatcher extends Sortable[ExprInstance] {
       case gi: GInt    => ScoredTerm(g, Leaves(Score.INT, gi.value)).pure[F]
       case gs: GString => ScoredTerm(g, Node(Score.STRING, Leaf(gs.value))).pure[F]
       case gu: GUri    => ScoredTerm(g, Node(Score.URI, Leaf(gu.value))).pure[F]
-      case EListBody(gl) =>
```

Comment:
Good move with the removal.
Actually, please incorporate the remains of the GroundSortMatcher into the ExprSortMatcher and delete the falltrough case in it, where the ground sort matcher was called.